cylc/flow/workflow_files.py

```diff
@@ -1907,7 +1907,8 @@ def get_source_workflow_name(source: Path) -> str:
     else the basename of the given path."""
```

Comment:
It would be worth mentioning in the docstring that the arg `source` must be an expanded (user and env vars) path